client/my-sites/themes/themes-selection.jsx

```diff
@@ -304,7 +303,7 @@ export const ConnectedThemesSelection = connect(
 		if ( source === 'wpcom' || source === 'wporg' ) {
 			sourceSiteId = source;
 		} else {
-			sourceSiteId = siteId && isJetpack ? siteId : 'wpcom';
+			sourceSiteId = siteId ? siteId : 'wpcom';
 		}
 
 		if ( isAtomic && ! hasUnlimitedPremiumThemes ) {
```

Comment:
[Boyscouting]
Is this even possible? Can we have Atomic sites that don't have the `WPCOM_FEATURES_PREMIUM_THEMES` feature? 🤔

Reply:
Yes, technically their subscription might be expired and for a week they could be on AT on free plan, until they get reverted back to simple.